services/QuillLMS/client/app/bundles/Shared/components/shared/teacherPreviewMenu.tsx

```diff
@@ -226,14 +224,6 @@ const TeacherPreviewMenuComponent = ({
   titleCards
 }: TeacherPreviewMenuProps) => {
 
-  React.useEffect(() => {
-    // we need to fetch grammar activities here
-    const activityUID = getParameterByName('uid', window.location.href);
-    if (activityUID) {
-      dispatch(getActivity(activityUID))
-    }
-  }, [session]);
-
```

Comment:
It's not going to be an issue for any of the other places that use this component if we remove it?